dan_layer/integration_tests/src/test_consensus.rs

```diff
@@ -644,6 +644,34 @@ async fn test_leader_fails_only_foreignly() {
     instance1.assert_shuts_down_safely().await;
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_if_hostuff_is_broken() {
```

Comment:
I lolled at the name, but I think we'll struggle to remember which part of hotstuff this is testing in a few months. Maybe `test_that_hotstuff_reaches_decide`

Reply:
A very useful and strangely simple test

Reply:
I've changed it to better reflects what's wrong with it.